Client/Ecosia/UI/WhatsNew/DataProvider/WhatsNewLocalDataProvider.swift

```diff
@@ -0,0 +1,83 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+
+/// A local data provider for fetching What's New items based on app version updates.
+final class WhatsNewLocalDataProvider: WhatsNewDataProvider {
+    
+    static let appVersionUpdateKey = "appVersionUpdateKey"
+        
+    /// The current app version provider is the DefaultAppVersionInfoProvider
+    /// from which the Ecosia App Version is retrieved
+    private let currentAppVersionProvider = DefaultAppVersionInfoProvider()
+    
+    /// Default initializer.
+    init() {}
+    
+    /// The items we would like to attempt to show in the update sheet
+    private let whatsNewItems: [Version: [WhatsNewItem]] = [
+        Version("9.0.0")!: [
+            WhatsNewItem(imageURL: URL.localURLForImageset(name: "tree", withExtension: "pdf"),
```

Comment:
following up on my comment from above. Instead of `imageURL` this could be `data`. It probably would make the whole getter async though.